.ci/update-upstream.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+upstreamRef="${1:-main}"
+
+upstream_base_url="https://raw.githubusercontent.com/open-telemetry/opentelemetry-java-instrumentation/${upstreamRef}"
+project_root="$(dirname "${0}")/.."
+
+upstream_version() {
+    set +e
+    curl -s "${upstream_base_url}/${1}" \
+        | grep "val.*${2}.*=" \
+        | sed 's/.*=//' \
+        | tr -d '"' \
+        | tr -d '[:blank:]' \
+        || echo 'unknown'
+    set -e
+}
+
+upstreamAgentAlphaVersion="$(upstream_version 'version.gradle.kts' alphaVersion)"
+upstreamContribVersion="$(upstream_version 'dependencyManagement/build.gradle.kts' otelContribVersion)"
+
+sed -i "s/^opentelemetryJavaagentAlpha = \".*/opentelemetryJavaagentAlpha = \"${upstreamAgentAlphaVersion}\"/" "${project_root}/gradle/libs.versions.toml"
+sed -i "s/^opentelemetryContribAlpha = \".*/opentelemetryContribAlpha = \"${upstreamContribVersion}\"/" "${project_root}/gradle/libs.versions.toml"
```

Comment:
```suggestion
sed -i '' -e "s/^opentelemetryJavaagentAlpha = \".*/opentelemetryJavaagentAlpha = \"${upstreamAgentAlphaVersion}\"/" "${project_root}/gradle/libs.versions.toml"
sed -i '' -e "s/^opentelemetryContribAlpha = \".*/opentelemetryContribAlpha = \"${upstreamContribVersion}\"/" "${project_root}/gradle/libs.versions.toml"
```

The original sed command fails on mac:
`sed: 1: "./.ci/../gradle/libs.ve ...": invalid command code .`